pkg/cmd/pr/pr.go

```diff
@@ -331,19 +331,18 @@ func (o *Options) SetCommitDetails(dir, commitMessage, commitTitle, application
 				} else if gitURL != "" {
 					if application == "" {
 						gitURLPart := strings.Split(gitURL, "/")
-						o.Application = gitURLPart[len(gitURLPart)-2] + "/" +
+						application = gitURLPart[len(gitURLPart)-2] + "/" +
 							strings.TrimSuffix(gitURLPart[len(gitURLPart)-1], ".git")
 					}
 					if commitMessage == "" {
 						o.CommitMessage = fmt.Sprintf("from: %s\n", gitURL)
 					}
-				}
-
-				if commitTitle == "" {
-					if application == "" {
-						o.CommitTitle = fmt.Sprintf("chore(deps): upgrade to version %s", o.Version)
-					} else {
-						o.CommitTitle = fmt.Sprintf("chore(deps): upgrade %s to version %s", o.Application, o.Version)
+					if commitTitle == "" {
+						if application == "" {
```

Comment:
As far as I can see the fix is changing so the variable used here is the one set above. But this fix breaks a few things:  o.Application is used elsewhere so not setting it here breaks those places.

Actually what you should do is to replace the content of SetCommitDetails to the code it had before your "[refactoring](https://github.com/jenkins-x-plugins/jx-updatebot/commit/406a99f8f60969f945babdcc5038f0b73be0860a)" (that introduced) this bug.